src/components/molecules/NavSearchBar/NavSearchResult.scss

```diff
@@ -0,0 +1,39 @@
+@import "scss/constants.scss";
+@import "./navSearchBarConstants.scss";
+
+.NavSearchResult {
+  cursor: pointer;
+  padding: $search-result-padding--vertical $search-result-padding--horizontal;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  align-content: space-between;
+
+  .user-avatar {
+    margin-right: $search-result-image--right-margin;
+  }
```

Comment:
[NB] I'm guessing this styling is to override `UserAvatar`'s styles? I wonder if we should refactor it to add `containerClassNames`/`imageClassNames`/etc props like we have done in other places?

Reply:
added class names props to `UserAvatar`:  https://github.com/sparkletown/sparkle/pull/1328/commits/ad90d7c37cc2e578794b1c7491b5b3c464017038